test/spec/algoliasearch.helper/queryID.js

```diff
@@ -21,7 +21,7 @@ test('the queryid should keep increasing when new requests arrives', function(t)
   t.end();
 });
 
-test('the response handler should check that the query is not outdated', function(t) {
+test.skip('the response handler should check that the query is not outdated', function(t) {
```

Comment:
I think that what you meant @vvo back then was "does it implement the outdated mechanism?" And the answer is yes (and the test is now available!!)